cli/command/manifest/create_list.go

```diff
@@ -34,21 +33,18 @@ func newCreateListCommand(dockerCli command.Cli) *cobra.Command {
 }
 
 func createManifestList(dockerCli command.Cli, args []string, opts annotateOpts) error {
-	// Just do some basic verification here, and leave the rest for when the user pushes the list
 	newRef := args[0]
-	targetRef, err := reference.ParseNormalizedNamed(newRef)
+	targetRef, err := normalizeReference(newRef)
 	if err != nil {
 		return errors.Wrapf(err, "error parsing name for manifest list (%s): %v", newRef)
 	}
+
+	// TODO: why is this here?
 	_, err = registry.ParseRepositoryInfo(targetRef)
 	if err != nil {
 		return errors.Wrapf(err, "error parsing repository name for manifest list (%s): %v", newRef)
 	}
 
-	// Check locally for this list transaction before proceeding
-	if _, isDigested := targetRef.(reference.Canonical); !isDigested {
-		targetRef = reference.TagNameOnly(targetRef)
-	}
 	manifestStore := newDefaultStore()
 	list, err := manifestStore.GetList(targetRef.String())
```

Comment:
Going back and forth between commits might be causing me to remember things incorrectly, but i think that somewhere I saw that this looked like it was going to be an issue because the store's Get func was called before the tag had been checked. Now I'll have to see if I find that again.